### PR TITLE
Tests for Shoov NodeJS server 

### PR DESCRIPTION
**Pull request by solker**
#15

### CODE REVIEW

**Comment by solker** [comment]
@amitaibu done, need tests for another routers?

**Comment by amitaibu** [comment]
Next is to connect to travis

**Comment by solker** [comment]
child-process-promise doesn't comparable with node v0.8
dockerode doesn't comparable with node v0.9
thus we will use minimum node v0.10 and next.

File: test/test.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    http.get(url, function (res) {
+      assert.equal(200, res.statusCode);
+      done();
+    });
+  });
+
+  it('should return an encrypted value', function(done) {
+    var options = {
+      privateKey: 'Y2DCJ6zGnVvZamcusAuNs1HfC-4AqDU46rvMgRAclSM',
+      keyToConvert: 'GITHUB_DUMMY_USERNAME',
```
Comment (amitaibu) [inline test/test.js]: lets use the common `foo` `bar` for key and value.

**Comment by solker** [comment]
@amitaibu ready for code review.

**Comment by amitaibu** [comment]
Thanks, please also add the Travis icon to the README

**Comment by solker** [comment]
@amitaibu done.

File: test/test.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+var server = http.createServer(app);
+
+describe('Shoov NodeJS Server', function() {
+
+  before(function () {
+    server.listen(port);
+  });
+
+  it('should running', function(done) {
```
Comment (amitaibu) [inline test/test.js]: should running => should be running

**Comment by amitaibu** [comment]
@solker please revert last commit from this PR, as it should be only about tests.

**Comment by amitaibu** [comment]
> please revert last commit from this PR, as it should be only about tests.

I'll do it, as I'd like to merge this.

**Comment by amitaibu** [comment]
Merged, thanks.

**Pull request closed without merging**